subsys/net/l2/wifi/wifi_shell.c

```diff
@@ -2852,15 +2852,77 @@ SHELL_STATIC_SUBCMD_SET_CREATE(
 );
 #endif /* CONFIG_WIFI_NM_WPA_SUPPLICANT_DPP */
 
-SHELL_STATIC_SUBCMD_SET_CREATE(wifi_commands,
-	SHELL_CMD_ARG(version, NULL, "Print Wi-Fi Driver and Firmware versions\n",
-		  cmd_wifi_version,
-		  1, 0),
-	SHELL_CMD(ap, &wifi_cmd_ap, "Access Point mode commands.\n", NULL),
-	SHELL_CMD_ARG(connect, NULL,
+static int cmd_wifi_ap(const struct shell *sh, size_t argc, char *argv[])
```

Comment:
Are those dummy command handlers needed? We seem to pass NULL as a handler in a few of the net shell commands, I hope that works fine?
https://github.com/zephyrproject-rtos/zephyr/blob/029540abece01361f52025534594a0b333cf0afd/subsys/net/lib/shell/cm.c#L952

Reply:
Ah, that could help, will try.

Reply:
Yep, the extra function helper are not needed, the beast works just fine without them.